scripts/updateReleaseNotes/releases.ts

```diff
@@ -1,4 +1,3 @@
-import { Octokit } from '@octokit/rest';
```

Comment:
lot of things changed since v16 including types... those types moves are not very well documented, so I did the best I could.  If we are no 100% confident we can drop this and use v16. 

in general it would be great to keep up to date with deps that we use for crucial parts of our pipelines.